extension/CHANGELOG.MD

```diff
@@ -1,4 +1,59 @@
-#v1.2.0 **Bug Fixes:**
+# Release notes - Version 2.2.0
+
+### Bug
+
+Ajustments to the pull up menu
```

Comment:
nit: can we include the bullet points like earlier releases? I think it does it make it easier to read

Reply:
lolol Jira takes them out when you take out the key! Whyyyyyyyy???

I'll add em back in